src/blockdev.rs

```diff
@@ -519,6 +520,32 @@ pub struct SavedPartitions {
     partitions: Vec<(u32, GPTPartitionEntry)>,
 }
 
+fn translate_mbr_types_to_gpt(sys: u8) -> Uuid {
+    // non inclusive best effort mapping of MBR types to GPT types
+    match sys {
+        // Linux filesystem
+        0x01 => uuid!("0FC63DAF-8483-4772-8E79-3D69D8477DE4"),
+        // Linux Filesystem Data
+        0x83 => uuid!("0FC63DAF-8483-4772-8E79-3D69D8477DE4"),
+        // Linux LVM
+        0x8e => uuid!("E6D6D379-F507-44C2-A23C-238F2A3DF928"),
+        // Linux Swap
+        0x82 => uuid!("0657FD6D-A4AB-43C4-84E5-0933C84B4F4F"),
+        _ => uuid!("00000000-0000-0000-0000-ffff00000000"),
```

Comment:
I'm having some hard time cross-checking where this "generic partition type UUID" is coming from. Did you invent it or sourced it from somewhere?

Reply:
LOL I thought I copied it from the wiki page, but I am not unsure It should be just an empty GUID as its not known.